doc/publishing_documentation.md

```diff
@@ -0,0 +1,249 @@
+# umati Dashboard
+
+Documentation of the topic and payload for `umati/v2/`
+
+Gateway: <https://github.com/umati/Dashboard-OPCUA-Client/tree/development>
+Server: <https://github.com/umati/Sample-Server/blob/develop/MachineTools/ShowcaseMachineTool.cpp>
+OPC UA Machine instance: NamespaceUri <http://example.com/ShowcaseMachineTool/>
+
+## Topics and payload
+
+### Overall topic
+
+`umati/v2/#`
+
+### Company and client spefic topic
+
+`umati/v2/<company>/<client>/`
+no payload
+
+### Online detection topic
```

Comment:
Is `umati/v2/<company>/<client>/clientOnline` the Last-Will-Topic?

https://cedalo.com/blog/mqtt-last-will-explained-and-example/

Reply:
Yes
https://github.com/umati/Dashboard-OPCUA-Client/blob/fde2534639113d0ddae618f9c9327285dc40f4ce/MqttPublisher_Paho/MqttPublisher_Paho.cpp#L91-L97